core/src/test/java/io/questdb/test/griffin/GroupByTest.java

```diff
@@ -2545,21 +2546,35 @@ public void testSelectDistinctOnAliasedColumnWithOrderBy() throws Exception {
                     "GROUP BY tab.created " +
                     "ORDER BY tab.created";
 
-            assertPlanNoLeakCheck(
-                    query,
-                    "Radix sort light\n" +
-                            "  keys: [ref0]\n" +
-                            "    Distinct\n" +
-                            "      keys: ref0\n" +
-                            "        VirtualRecord\n" +
-                            "          functions: [created]\n" +
-                            "            Async JIT Group By workers: 1\n" +
-                            "              keys: [created]\n" +
-                            "              filter: null!=created\n" +
-                            "                PageFrame\n" +
-                            "                    Row forward scan\n" +
-                            "                    Frame forward scan on: tab\n"
-            );
+            if (JitUtil.isJitSupported()) {
```

Comment:
nit: `assertPlanNoLeakCheck` already includes a `JitUtil.isJitSupported()` check, so that there should be no point in having if/else branches:
```java
            if (!JitUtil.isJitSupported()) {
                expectedPlan = Chars.toString(expectedPlan).replace("Async JIT", "Async");
            }
```

Reply:
something was explicitly failing, i'll double check

Reply:
removed complexity here: https://github.com/questdb/questdb/pull/5445/commits/d9524e8749e659c8740ed5a22dfadbb38e5d6b0e